runtime/onert/core/src/exec/Executors.cc

```diff
@@ -149,9 +149,21 @@ void Executors::checkSupportedMultimodel() const
         throw std::runtime_error{"Cannot find 1st model's input buffer"};
     }
   }
+
+  // Assumption: nnpkg inputs/outputs and edges are not duplicated.
+  for (const auto &edge : _model_edges->edges)
+  {
+    if (std::find(_model_edges->pkg_inputs.begin(), _model_edges->pkg_inputs.end(), edge.to) !=
+          _model_edges->pkg_inputs.end() ||
+        std::find(_model_edges->pkg_outputs.begin(), _model_edges->pkg_outputs.end(), edge.from) !=
+          _model_edges->pkg_outputs.end())
+    {
+      throw std::runtime_error{
+        "Multi model execution does not support duplicating nnpkg inputs/outputs with edges yet"};
+    }
+  }
```

Comment:
Is it possible case when edge's end is nnpkg's input? (maybe possible edge's begin is pkg's out)

Reply:
Does `edge's end` you mentioned means `to` in an edge?

Reply:
Did you mean that `from` in edge could be an output of nnpkg?
It's possible.

| possibility of duplication | Edge `to` (input of subgraph) | Edge `from` (output of subgraph) |
| :---: | :---: | :---: |
| nnpkg input (input of subgraph) | X (*1) | X (*2) |
| nnpkg output (output of subgraph) | X (*2) | O |

*1. The subjects who determine values of each buffer are different.
  - nnpkg input : user input
  - Edge `to` : output of another subgraph

*2. `IOIndex` of inputs and outputs of subgraph is distinct.

Reply:
My comment is about `std::find(_model_edges->pkg_inputs.begin(), _model_edges->pkg_inputs.end(), edge.to) != _model_edges->pkg_inputs.end()`.

Reply:
My comment is about `std::find(_model_edges->pkg_inputs.begin(), _model_edges->pkg_inputs.end(), edge.to)`.

Reply:
> My comment is about std::find(_model_edges->pkg_inputs.begin(), _model_edges->pkg_inputs.end(), edge.to). I thought it's impossible case.

I see. I will separate impossible cases from possible cases and throw those two exceptions separately.

Reply:
Impossible cases should check on package loading phase, IMHO.